systems/node/notify/internal/server/event.go

```diff
@@ -21,68 +23,176 @@ import (
 	mb "github.com/ukama/ukama/systems/common/msgBusServiceClient"
 	epb "github.com/ukama/ukama/systems/common/pb/gen/events"
 )
+ 
+ type NotifiyEventServer struct {
```

Comment:
And we still keep the Event server related code.

Reply:
Yes, because we are going to handle node events as discussed.

Reply:
The event server code was not pushed my bad.it's fixed now.